Makefile

```diff
@@ -133,8 +133,8 @@ build-reproducible-generic: go.sum
 		--build-arg VERSION="$(VERSION)" \
 		-f Dockerfile .
 	$(DOCKER) create -ti --name $(subst /,-,latest-build-$(PLATFORM)) latest-build-$(PLATFORM) kid
-	mkdir -p $(CURDIR)/build/$(NETWORK)/$(PLATFORM)/
```

Comment:
Why remove the build dir entirely ? 

Reply:
Everything is now built at the root for the shasum to display correct path.
I was highly inspired by horcrux's setup cf: https://github.com/strangelove-ventures/horcrux/blob/03f565d67bb5394a4b3017a1adb00561d5297724/.github/workflows/release.yml#L63